service/eventer/github/client.go

```diff
@@ -277,7 +277,8 @@ func (e *GithubEventer) postDeploymentStatus(project string, id int, state deplo
 	}
 
 	if resp.StatusCode != http.StatusCreated {
-		return microerror.Maskf(unexpectedStatusCode, fmt.Sprintf("received non-200 status code: %v", resp.StatusCode))
+		body, _ := ioutil.ReadAll(resp.Body)
+		return microerror.Maskf(unexpectedStatusCode, fmt.Sprintf("received non-200 status code: %v, body: %q", resp.StatusCode, string(body)))
```

Comment:
Please apply the same fix above. I think lines 156 and 221.

Reply:
Done, PTAL